dash/dash.py

```diff
@@ -339,16 +339,21 @@ def _generate_config_html(self):
         ).format(json.dumps(self._config()))
 
     def _generate_meta_html(self):
+        has_ie_compat = any(
+            x.get('http-equiv', '') == 'X-UA-Compatible'
+            for x in self._meta_tags)
         has_charset = any('charset' in x for x in self._meta_tags)
 
         tags = []
+        if not has_ie_compat:
+            tags.append('<meta equiv="X-UA-Compatible" content="IE=edge">')
```

Comment:
Hmm. Should this not be `http-equiv="..."`?

Reply:
Yes, it should.